packages/models-library/src/models_library/api_schemas_webserver/resource_usage.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+from enum import Enum
+
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from models_library.services import ServiceKey, ServiceVersion
+from pydantic import BaseModel
+
+# Frontend API
+
+
+class ContainerStatus(str, Enum):
+    RUNNING = "running"
+    FINISHED = "finished"
+
+
+class ContainerGet(BaseModel):
+    project_uuid: ProjectID
+    project_name: str | None
+    node_uuid: NodeID
+    node_label: str | None
+    service_key: ServiceKey
+    service_version: ServiceVersion
+    start_time: datetime
+    duration: float
+    processors: float
+    core_hours: float
+    status: ContainerStatus
+
+    class Config:
+        validation = False
```

Comment:
Q: interesting ... what is the use of this if it is not validated? I guess all responses payloads are pre-validated!? If so, I would put this Config in the webserver. Think that this model is to be used by the client side as well

Reply:
you are right, i will remove it in upcoming PR :+1: